src/gui/settings.cpp

```diff
@@ -530,6 +530,16 @@ void FurnaceGUI::drawSettings() {
         }
         ImGui::Unindent();
 
+#ifdef IS_MOBILE
+        // SUBSECTION HAPTIC
+        CONFIG_SUBSECTION("Vibration");
+
+        ImGui::Indent();
+        if (ImGui::SliderFloat("Strength",&settings.vibrationStrength,0.0f,1.0f)) settingsChanged=true;
+        if (ImGui::SliderInt("Length",&settings.vibrationLength,50,500)) settingsChanged=true;
```

Comment:
Perform range check here. Make sure the user isn't able to exceed the slider range.

Minimum length should be 10ms, not 50.

Reply:
just the length slider or both?

Reply:
Both.